tests/bip39.rs

```diff
@@ -17,33 +16,39 @@ struct TestVector {
     seed: &'static str,
 }
 
+#[test]
+fn test_separator() {
+    use unicode_normalization::UnicodeNormalization;
+    let s = "　";
+    // assert!(is_nfkd(s));
```

Comment:
What's up with this?

Reply:
Fixed that. `s` contains space in non-normalized form.